doc/rtd/tutorial/index.rst

```diff
@@ -3,47 +3,35 @@
 Tutorials
 *********
 
-This section contains step-by-step tutorials to help you get started with
-``cloud-init``. We hope our tutorials make as few assumptions as possible and
-are accessible to anyone with an interest in ``cloud-init``. They should be a
-great place to start learning about ``cloud-init``, how it works, and what it's
-capable of.
+Our step-by-step tutorials will help you learn about cloud-init and what it can
+do.
 
------
+New user tutorial
+=================
 
-Core tutorial
-=============
+If you are completely new to cloud-init and would like a more thorough
+introduction, we suggest starting with the
+:ref:`new user tutorial <tutorial_qemu>`.
 
-This tutorial, which we recommend if you are completely new to ``cloud-init``,
-uses the QEMU emulator to introduce you to all of the key concepts, tools,
-processes and operations that you will need to get started.
+This tutorial uses the QEMU emulator to introduce you to all of the key
+concepts, tools, processes and operations that you will need to use cloud-init
+successfully.
 
-.. toctree::
-   :maxdepth: 1
+Further tutorials
+=================
 
-   qemu.rst
+This tutorial is recommended if you have some familiarity with cloud-init's key
+concepts already. It uses LXD containers to show more of cloud-init's
+capabilities.
 
-Quick-start tutorial
-====================
+* :ref:`Part 1: quick deployment <tutorial_lxd>`
```

Comment:
A part 1 without a part 2 feels odd. Were you planning to add more here?

Reply:
I am yes, but it won't be part of this PR because it's still in the early draft stage.

Reply:
It's been agreed that Part 2 won't be done any time soon - would you prefer me to remove the reference, or should I leave it like this as a placeholder for where any future tutorials should go?